solr/src/obj/template_data.coffee

```diff
@@ -389,11 +389,16 @@ window.google_templates =
             $(document).trigger('maybe_update_state',{ q: $(this).val(), page: 1 })
     postproc: (el,data) ->
       $(document).on 'maybe_update_state', (e,change,incr) ->
-        $.getJSON "/Multi/Ajax/psychic",{ q: change.q ? '' }, (data) ->
+        facet_species = data?.state?.hub?.params?.facet_species
```

Comment:
AFAIK the `template_data.coffee` is the main file and when you run a build file, it will generate the `template_data.js` file. @ens-ds23  can confirm this. 

I will also avoid using the ES6 syntax in here. it might get into the .js file which will cause the js minifier to fail.

Reply:
>  will also avoid using the ES6 syntax in here

This is not ES6 syntax; it is coffeescript syntax; there are quite a few examples of optional chaining in our coffeescript files.